### ISSUE TITLE
Error when closing a console

### ISSUE THREAD

**Issue by homocomputeris** [spyder-ide/spyder#6027]
## Description

**What steps will reproduce the problem?**

Try to close console when some code is running

**What is the expected output? What do you see instead?**
Close the console tab. Got a freeze

**Please provide any additional information below**
This is an automatic report by Spyder

  File "/usr/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 906, in <lambda>
    clear_variables))
  File "/usr/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 948, in execute_code
    sw.sig_prompt_ready.disconnect()
TypeError: disconnect() failed between 'sig_prompt_ready' and all its connections

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.10.0, PyQt5 5.9.2 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.21.0 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.5 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.11.1 (OK)
psutil >=0.3     :  5.4.1 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.27.3 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.2.1 (OK)
pylint >=0.25    :  1.7.5 (OK)

```

**Comment by CAM-Gerlach** [comment]
Thanks for reporting. Unfortunately, I was unable to reproduce this on Win 8.1 with QT 5.6 and iPython 6.1.0. I tried
```
import time
time.sleep(20)
```
and then closing the console, and while there was a brief hang of a couple seconds, it successfully closed and no error occurred. Is there particular code this occurs with (e.g. parallelized? I've had Spyder hang before in such situations). Does the above code produce the problem for you? Is there anything else particular about your environment or the circumstances during which this occurs? Please let us know as soon as you can; otherwise this issue will likely eventually get closed.

**Comment by ccordoba12** [comment]
@CAM-Gerlach, this error is fixed already and it'll be part of our 3.2.5 version.

**Comment by CAM-Gerlach** [comment]
Not sure how I missed #5748 , sorry. Thanks for the catch.
